packages/devtools_app/lib/src/memory/memory_heap_tree_view.dart

```diff
@@ -456,11 +452,72 @@ class HeapTreeViewState extends State<HeapTree>
   }
 
   Widget buildSnapshotTables(Widget snapshotDisplay) {
+    final themeData = Theme.of(context);
+
+    if (snapshotDisplay == null) {
+      // Display help text about how to collect data.
+      return Column(
```

Comment:
just use a Center widget instead of using this column with alignment center

Reply:
Done.